### PR TITLE
bump versions for runc, containerd and etcd

### PR DESCRIPTION
**Pull request by ncopa**
- runc 1.0.0-rc95 (CVE-2021-30465)
- containerd 1.4.5
- etcd 3.4.16
- konnectivity 0.0.17
- kubernetes 1.21.1 (CVE-2021-25737)

update kube-proxy container version to match kubernetes and add a unit test for that.

### CODE REVIEW

**Comment by ncopa** [comment]
passes network conformance tests https://github.com/k0sproject/k0s/actions/runs/834954070

**Review by jasmingacic — DISMISSED** [review]

**Comment by ncopa** [comment]
conformance test passed: https://github.com/k0sproject/k0s/actions/runs/856922799

File: embedded-bins/Makefile.variables [hunk 1]
```diff
@@ -1,9 +1,9 @@
-runc_version = 1.0.0-rc93
-containerd_version = 1.4.4
-kubernetes_version = 1.21.0
+runc_version = 1.0.0-rc95
```
Comment (s0j) [inline embedded-bins/Makefile.variables:1]: The PR description says `rc94`, is this right?

Comment (ncopa) [reply embedded-bins/Makefile.variables:1]: yeah. when PR was created it was `-rc94`but there is a recent CVE so I added a another commit with `-rc95` in the same PR

Comment (ncopa) [reply embedded-bins/Makefile.variables:1]: I updated PR description so it should match

**Review by s0j — APPROVED** [review]

**Pull request merged**